src/cli/utils/restore_config.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/spf13/cobra"
+
+	"github.com/alcionai/corso/src/cli/flags"
+	. "github.com/alcionai/corso/src/cli/print"
+	"github.com/alcionai/corso/src/internal/common/dttm"
+	"github.com/alcionai/corso/src/pkg/control"
+)
+
+type RestoreCfgOpts struct {
+	Collisions  string
+	Destination string
+
+	Populated flags.PopulatedFlags
+}
+
+func makeRestoreCfgOpts(cmd *cobra.Command) RestoreCfgOpts {
+	return RestoreCfgOpts{
+		Collisions:  flags.CollisionsFV,
+		Destination: flags.DestinationFV,
+
+		Populated: flags.GetPopulatedFlags(cmd),
```

Comment:
This needs a comment about what it means. Does it mean there was a value for the flag at all (even if it was the default), the user gave a value (even if it matched the default), or the user gave a value different from the default?